vbscripts/popUp.js

```diff
@@ -0,0 +1,207 @@
+const project = [
+  {
+    projectTitle: 'Profesional Art Printing Data',
```

Comment:
- [ ] Great job so far. like the previous reviewer noted, kindly ensure the mobile popup is full height like that of the figma version. 
- [ ] please make your test and seelive/seesource buttons align with the image.
your version
![Screenshot (267)_LI](https://user-images.githubusercontent.com/65298048/180243202-798a3754-460b-4300-847f-a0e983e420f1.jpg)
figma version
![Screenshot (266)](https://user-images.githubusercontent.com/65298048/180243358-f7861e47-4afc-4118-a9d5-2a9deae90257.png)